pkg/api/validation/validation_test.go

```diff
@@ -1129,7 +1143,19 @@ func TestValidateVolumes(t *testing.T) {
 			},
 		},
 		{
-			name: "valid Secret with projection",
+			name: "valid Secret with defaultMode",
+			vol: api.Volume{
+				Name: "secret",
+				VolumeSource: api.VolumeSource{
+					Secret: &api.SecretVolumeSource{
+						SecretName:  "my-secret",
+						DefaultMode: newInt32(0644),
+					},
+				},
+			},
+		},
+		{
```

Comment:
No, added this and the case with projection now is with projection and mode. The diff is shown in a tricky rate